content/en/docs/started/workstation/getting-started-windows.md

```diff
@@ -16,31 +12,44 @@ In addition, with the recent announcement of
 some of the [linux installation options](/docs/started/workstation/getting-started-linux)
```

Comment:
There is some documentation on Kubeflow on WSL2, however it's still an overkill, not providing easier access. I'd remove this suggestion and when it becomes easier we can drive users in that direction.